package.json

```diff
@@ -88,6 +89,7 @@
     "react-markdown": "^8.0.0",
     "react-redux": "^7.2.6",
     "react-show-more-text": "^1.6.2",
+    "react-table": "^7.8.0",
```

Comment:
Why another non needle dependency? We are trying to reduce imports

Reply:
As far as I am aware, react-table is necessary to implement pagination in the table which displays prompts/cypher queries. I haven't found a needle equivalent

Reply:
Worst case scenario, there's @mui/x-data-grid. However, I do rather prefer using https://storybook-components-build.appspot.com/?path=/docs/tables-how-to-use--docs We will be soon removing x-data-grid.

We will not add any new style lib @alfredorubin96

Reply:
Agree lets not add `@tanstack/react-table` and `react-table`